cypress/integration/pages/testsForAllAMPPages.js

```diff
@@ -14,6 +19,23 @@ export const testsThatFollowSmokeTestConfigForAllAMPPages = ({
       it('should have an AMP attribute on the page', () => {
         cy.get('html').should('have.attr', 'amp');
       });
+      describe('ATI', () => {
```

Comment:
This is going to run for ALLLL services, instead limit it to where there are differences.
I would write if smoke global var is true then run logic that's here, else run for only 1 PS, 1 WS and 1 GNL service.